pkg/commands/validate/validate_test.go

```diff
@@ -16,52 +16,50 @@ package validate
 
 import (
 	"context"
-	"errors"
 	"strings"
 	"testing"
 
+	"k8s.io/apimachinery/pkg/util/validation/field"
+
 	bundle "github.com/GoogleCloudPlatform/k8s-cluster-bundle/pkg/apis/bundle/v1alpha1"
 	"github.com/GoogleCloudPlatform/k8s-cluster-bundle/pkg/commands/cmdlib"
 	"github.com/GoogleCloudPlatform/k8s-cluster-bundle/pkg/testutil"
 )
 
 type fakeBundleValidator struct {
-	errs []error
+	errs field.ErrorList
 }
 
-func (f *fakeBundleValidator) Validate() []error {
+func (f *fakeBundleValidator) Validate() field.ErrorList {
 	return f.errs
 }
 
 func TestRunValidate(t *testing.T) {
```

Comment:
Another test you can do, which will quickly give you some more coverage, is to try to trigger every error in your validation, and make sure the total error count matches what you expect (it's kind of the opposite of a "set every field to something valid and ensure I get no errors" test).

Reply:
Added